Header header     # Original sensor_msgs/Image header
uint8[] data      # x264 NAL unit
int32 img_width   # Image width
int32 img_height  # Image height


